be/src/storage/lake/tablet_reader.cpp

```diff
@@ -121,6 +133,78 @@ Status TabletReader::init_predicates(const TabletReaderParams& params) {
     return Status::OK();
 }
 
+Status TabletReader::init_delete_predicates(const TabletReaderParams& params, DeletePredicates* dels) {
```

Comment:
If you reimplement the interface, can we reuse the predicates of query engine?

Reply:
Yes, it is for lake tablet.